src/webgldrawer.js

```diff
@@ -486,11 +485,18 @@
 
         // Public API required by all Drawer implementations
         /**
-        * Required by DrawerBase, but has no effect on WebGLDrawer.
-        * @param {Boolean} enabled
+        * Sets whether image smoothing is enabled or disabled
+        * @param {Boolean} enabled If true, uses gl.LINEAR as the TEXTURE_MIN_FILTER and TEXTURE_MAX_FILTER, otherwise gl.NEAREST.
         */
         setImageSmoothingEnabled(enabled){
-            // noop - this property does not impact WebGLDrawer
+            const changed = this._imageSmoothingEnabled !== enabled;
```

Comment:
Why not just return if it's not a change? We shouldn't need to redraw the world in that case.

Reply:
Good point, I've just put all the logic inside a conditional.